docs/core-concepts/testing.md

```diff
@@ -1,19 +1,63 @@
 ---
 layout: doc
-title: Testing
+title: Testing (Everest)
 ---
 
 # Testing
 
-In the case you don't have a charger that supports OCPP 2.0.1 to experiment with, we can recommend using the Linux Foundation Energy project EVerest.
-[See here](https://github.com/EVerest) for the repository.
-They have built an open source version of charger firmware and also allow for using it as a simulator.
-They support OCPP 2.0.1 which makes it a great testing opportunity with CitrineOS.
-For the long route of setting up EVerst you can follow their documentation and build the project yourself.[See here for Docs](https://everest.github.io/latest/general/03_quick_start_guide.html)
+In the case you don't have a charger that supports OCPP 2.0.1 to experiment with, we can recommend using the Linux 
+Foundation Energy project EVerest. [See here](https://github.com/EVerest) for the repository.  They have built an open source version of
+charger firmware and also allow for using it as a simulator. They support OCPP 2.0.1 which makes it a great testing 
+opportunity with CitrineOS. For the long route of setting up EVerst you can follow their documentation and build 
+the project yourself.[See here for Docs](https://everest.github.io/latest/general/03_quick_start_guide.html)
 
-As a short-cut you can also use their demo repository that hosts a Docker packaged EVerest image. [See here for Github Repo](https://github.com/EVerest/everest-demo)
+# Running Everest
+In order to alleviate some of the complexities that may arise when starting Everest, we have created
+some helpful commands that should help in getting the Everest charger simulator running locally and targeting
+CitrineOS.
 
-If you want to run EVerest on the side while developing and making changes, you can follow the steps below.
+You will notice in `/Server/everest` directory the new files created to support running Everest within Docker.
+In addition, we created some helpful NPM commands:
+
+- `npm run start-everest`
+- and
+- `npm run start-everest-windows`
+
+Both of which in essence do the same thing which is to trigger the `docker compose up` command from within
+the `/Server/everest` directory so that it can pick up the `Dockerfile` and the `docker-compose.yml` files:
+
+```shell
+EVEREST_IMAGE_TAG=0.0.16 EVEREST_TARGET_URL=ws://host.docker.internal:8081/cp001 docker-compose up
```

Comment:
Honestly, you can probably just get rid of this code snippet, and then maybe adjust the wording below to read as:

"In these NPM start commands, you will notice that there are two args..."

Reply:
hmm, my initial thought was to include this in here explicitly showing them that they can also just run this command directly without having to use npm, maybe it is helpful to see?

Reply:
But this code snippet isnt a command you can run?

Reply:
Oh wait, didn't see the docker compose up at the end